cmd/internal/cli/action_flags.go

```diff
@@ -37,6 +37,7 @@ var (
 	singularityEnvFile string
 	noMount            []string
 	proot              string
+	device             []string
```

Comment:
`devices`?

Reply:
Definitely just `device` here at present please. We are matching the existing pattern for other flag vars where the var is named the same as the flag, which is non-plural `--device`.

If we change it to plural (for which I see the argument, and tend to agree), we'll do that consistently across our other flag vars too, in a separate re-factoring.